### PR TITLE
Lps 137135

### PR DESCRIPTION
**Pull request by DL-EthanSun**

### CODE REVIEW

**Comment by DL-EthanSun** [comment]
ci:test:project-templates

**Comment by DL-EthanSun** [comment]
ci:test:project-templates

**Comment by DL-EthanSun** [comment]
ci:test:project-templates

**Comment by DL-EthanSun** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-rest-builder/src/test/java/com/liferay/project/templates/rest/builder/ProjectTemplatesRESTBuilderWorkspaceTest.java [hunk 1]
```diff
@@ -68,9 +69,14 @@
...
 				{"backend-integration", "com.liferay.docs.guestbook", "7.3.7"},
+				{
+					"backend-integration", "com.liferay.docs.guestbook",
+					"7.4.1-1"
+				},
 				{"sample", "com.test.sample", "7.1.3-1"},
 				{"sample", "com.test.sample", "7.2.1-1"},
-				{"sample", "com.test.sample", "7.3.7"}
+				{"sample", "com.test.sample", "7.3.7"},
+				{"sample", "com.test.sample", "7.4.1-1"}
```
Comment (lawrence-lee) [inline modules/sdk/project-templates/project-templates-rest-builder/src/test/java/com/liferay/project/templates/rest/builder/ProjectTemplatesRESTBuilderWorkspaceTest.java:79]: Latest 7.4 GA is 7.4.2-1?

File: modules/build.gradle [hunk 2]
```diff
@@ -747,4 +747,4 @@ private void _updateGradleSourcePaths(String gradleVersion, EclipseModel eclipse
 				}
 			}
 		})
-}
\ No newline at end of file
+}
```
Comment (lawrence-lee) [inline modules/build.gradle:750]: Extra whitespace?

**Comment by DL-EthanSun** [comment]
ci:test:project-templates

File: modules/build.gradle (hunk 2 above)
Comment (DL-EthanSun) [reply modules/build.gradle]: Sorry, I use vi to edit file, it added, and I didn't check and find it.

**Comment by DL-EthanSun** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
Just started reviewing :)

**Pull request closed without merging**